doc/api/permissions.md

```diff
@@ -126,6 +126,43 @@ does not exist, the wildcard will not be added, and access will be limited to
 yet, make sure to explicitly include the wildcard:
 `/my-path/folder-do-not-exist/*`.
 
+#### Using the Permission Model with `npx`
+
+If you're using [`npx`][] to execute a Node.js script, you can enable the
+Permission Model by passing the `--node-options` flag. For example:
+
+```bash
+npx --node-options="--permission" package-name
+```
+
+This sets the `NODE_OPTIONS` environment variable for all Node.js processes
+spawned by [`npx`][], without affecting the `npx` process itself.
+
+**FileSystemRead Error with `npx`**
+
+The above command will likely throw a `FileSystemRead` invalid access error
```

Comment:
`npx` is not what's requiring file system access, node itself is.


This will only error on executables that are shebanged node scripts, i.e. their first line is:

```sh
#!/usr/bin/env node
```

Your shell spawns `node` with the first parameter being this script in that case, and because that newly spawned `node` is has these permission args set (because npm/npx propagates them via process.env.NODE_OPTIONS when you use the `--node-options` flag), it then needs explicit permission to load that script.

```sh
$ cat example.js 
#!/usr/bin/env node
console.log(process.argv)

$ ./example.js 
[
  '/Users/wraithgar/.nvm/versions/node/v22.13.0/bin/node',
  '/Users/wraithgar/Development/scratch/binexample/example.js'
]
```